pyproject.toml

```diff
@@ -0,0 +1,55 @@
+[build-system]
+requires = ["setuptools >= 61.0", "Cython"]
```

Comment:
According to the [setuptools documentation](https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html), [python packaging user guide](https://packaging.python.org/en/latest/guides/writing-pyproject-toml/) and [setuptools changelog](https://setuptools.pypa.io/en/stable/history.html#v61-0-0) the minimal setuptools version supporting `pyproject.toml` configuration is `61.0`. It further requires `python>=3.7`. 

@mthh what is your stand on still supporting python3.6 (end of life on 2021-12-23)?

Reply:
II think it's now reasonable to drop Python 3.6.